lhotse/dataset/feature_transforms.py

```diff
@@ -1,15 +1,40 @@
 import torch
 
+from typing import Union
+
 from lhotse import CutSet
+from lhotse.utils import Pathlike
+
+
+class GlobalMVN(torch.nn.Module):
+    """Apply global mean and variance normalization"""
 
+    def __init__(self, feature_dim: int):
+        super().__init__()
+        self.register_buffer("norm_means", torch.zeros(feature_dim))
+        self.register_buffer("norm_stds", torch.ones(feature_dim))
 
-class Standardize:
-    def __init__(self, cuts: CutSet):
+    @classmethod
+    def from_cuts(cls, cuts: CutSet) -> "GlobalMVN":
```

Comment:
Please add a parameter `max_cuts`, for large datasets you probably don't need to go through everything to get a reasonable estimate (`compute_global_feature_stats()` supports it already).

Reply:
Great! adding